pkgs/os-specific/linux/kernel/update.sh

```diff
@@ -12,3 +12,6 @@ COMMIT=1 ./update-libre.sh || echo "update-libre failed with exit code $?"
 
 echo "Update linux-hardened"
 COMMIT=1 ./hardened/update.py || echo "update-hardened failed with exit code $?"
+
+echo Update linux-testing-bcacehfs
```

Comment:
How long do you think we still have this kernel? If the mainline merge goes through this would be just a matter of weeks?

Reply:
I just added it because it made my job simpler while writing this patch, I already had one sitting around, I assume bcachefs is coming in v6.7 which is a bit longer time schedule.

We can pull it out in to a `nur-kernels` as @RaitoBezarius suggested in the IRC.

Reply:
6.7 will be here in November FWIW.

Reply:
The issue now is that we are dropping 6.4 in #258569, so actually, it would be good to extract the meaningful part to bump to 6.5 and get it ready ASAP even before November.